tools/monorepo-utils/src/core/logger.ts

```diff
@@ -25,10 +25,19 @@ export class Logger {
 		] as number;
 	}
 
-	static error( message: string ) {
+	static error( err: unknown, exitOnFail = true ) {
```

Comment:
This allows us to continue the pattern we have now of not checking error type when calling `Logger.error`. It will do it's best to log the error based on the type (when catching in TS/JS the type of the error is always unknown due to the fact you can `throw` anything.